pkg/tnf/operator/starter.go

```diff
@@ -0,0 +1,177 @@
+			}
+
+			runTnfSetupJobController(ctx, controllerContext, operatorClient, kubeClient, kubeInformersForNamespaces)
+
+		}
+	}
+	return nil
+}
+
+func runExternalEtcdSupportController(ctx context.Context, controllerContext *controllercmd.ControllerContext,
+	operatorClient v1helpers.StaticPodOperatorClient, envVarController *etcdenvvar.EnvVarController,
+	kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces, configInformers configv1informers.SharedInformerFactory,
+	networkInformer v1.NetworkInformer, controlPlaneNodeInformer cache.SharedIndexInformer, kubeClient *kubernetes.Clientset) {
+
+	klog.Infof("starting external etcd support controller")
+	externalEtcdSupportController := externaletcdsupportcontroller.NewExternalEtcdEnablerController(
+		operatorClient,
+		os.Getenv("IMAGE"),
+		os.Getenv("OPERATOR_IMAGE"),
+		envVarController,
+		kubeInformersForNamespaces.InformersFor("openshift-etcd"),
+		kubeInformersForNamespaces,
+		configInformers.Config().V1().Infrastructures(),
+		networkInformer,
+		controlPlaneNodeInformer,
+		kubeClient,
+		controllerContext.EventRecorder,
+	)
+	go externalEtcdSupportController.Run(ctx, 1)
+}
+
+func runTnfResourceController(ctx context.Context, controllerContext *controllercmd.ControllerContext, kubeClient *kubernetes.Clientset, dynamicClient *dynamic.DynamicClient, operatorClient v1helpers.StaticPodOperatorClient, kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces) {
+	klog.Infof("starting Two Node Fencing static resources controller")
+	tnfResourceController := staticresourcecontroller.NewStaticResourceController(
+		"TnfStaticResources",
+		tnf_assets.Asset,
+		[]string{
+			"tnfdeployment/sa.yaml",
+			"tnfdeployment/role.yaml",
+			"tnfdeployment/role-binding.yaml",
+			"tnfdeployment/clusterrole.yaml",
+			"tnfdeployment/clusterrole-binding.yaml",
+		},
+		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient).WithDynamicClient(dynamicClient),
+		operatorClient,
+		controllerContext.EventRecorder,
+	).WithIgnoreNotFoundOnCreate().AddKubeInformers(kubeInformersForNamespaces)
+	go tnfResourceController.Run(ctx, 1)
+}
+
+func runTnfAuthJobController(ctx context.Context, nodeName string, controllerContext *controllercmd.ControllerContext, operatorClient v1helpers.StaticPodOperatorClient, kubeClient *kubernetes.Clientset, kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces) {
+	klog.Infof("starting Two Node Fencing auth job controller")
+	tnfJobController := tnflibgo.NewJobController(
+		"TnfJob-"+nodeName,
+		tnf_assets.MustAsset("tnfdeployment/authjob.yaml"),
+		controllerContext.EventRecorder,
+		operatorClient,
+		kubeClient,
+		kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Batch().V1().Jobs(),
+		[]factory.Informer{},
+		[]tnflibgo.JobHookFunc{
+			func(_ *operatorv1.OperatorSpec, job *batchv1.Job) error {
+				// set operator image pullspec
+				job.Spec.Template.Spec.Containers[0].Image = os.Getenv("OPERATOR_IMAGE")
```

Comment:
ah I see. Can you make the yaml files aware that this will be injected? add a comment or "<image>" so it doesn't inadvertently parse as an image we try to pull